base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -62,8 +63,12 @@ use crate::output_manager_service::{
 pub enum OutputManagerRequest {
     GetBalance,
     AddOutput((Box<UnblindedOutput>, Option<SpendingPriority>)),
+    // ToDo: This API request could probably be removed by expanding test utils if only needed for testing
+    AddRewindableOutput((Box<UnblindedOutput>, Option<SpendingPriority>, Option<RewindData>)),
```

Comment:
How does a developer know to use this and not `AddRewindableOutputWithTxId`?

Reply:
`AddRewindableOutput` is currently used in unit tests only, for example as in `async fn test_utxo_selection_no_chain_metadata()`.

`AddRewindableOutputWithTxId` is used in `pub async fn send_sha_atomic_swap_transaction` where `TxId` is chosen at the start of the protocol.